### ISSUE TITLE
multipart XML schemas with folders

### ISSUE THREAD

**Issue by philbarker** [t3-innovation-network/desm#423]
Is it possible to upload a multipart XML based schema where the XSDs are in sub-directories?

I know this is possible with JSON-Schema because we can upload a .zip archive.

**Comment by excelsior** [comment]
No, there's no such possibility yet, XML schemas can only be uploaded separately.

**Comment by philbarker** [comment]
OK, removing the question tag, but noting that this stands as a potential enhancement.
